package.json

```diff
@@ -27,5 +27,6 @@
         "url": "https://github.com/Microsoft/tslib.git"
     },
     "main": "tslib.js",
+    "jsnext:main": "tslib.es6.js",
```

Comment:
that is a very good point, [this issue](https://github.com/jsforum/jsforum/issues/5) seems to indicate that it should point to a file which is consumable by all engines for using es6 import/export statements.

In my scenario I actually do a second typescript transpilation to output the rollup bundle in system.register format and that removes the const keywords.

Shall I change the file to use ```export var```?